docs/nodes/nodes-library/nodes/CoinGecko/README.md

```diff
@@ -0,0 +1,107 @@
+---
+permalink: /nodes/n8n-nodes-base.coinGecko
+description: Learn how to use the CoinGecko node in n8n
+---
+
+# CoinGecko
+
+[CoinGecko](https://www.coingecko.com) provides a fundamental analysis of the crypto market. In addition to tracking price, volume and market capitalization, CoinGecko tracks community growth, open-source code development, major events and on-chain metrics.
+
```

Comment:
You can add an authentication section like [this](https://docs.n8n.io/nodes/n8n-nodes-base.hackerNews/) for consistency.